phone/src/ui/hooks/useContextMenu.tsx

```diff
@@ -12,9 +12,9 @@ export const MapStringOptions = (current, onClick) => (string) => {
   };
 };
 
-export interface SettingOption {
+export interface SettingOption<T = void> {
```

Comment:
This generic allows you to optionally specify valid values for the SettingOption